### ISSUE TITLE
feat(host): Setup Telemetry

### ISSUE THREAD

**Issue by refcell** [ethereum-optimism/kona#134]
**Description**

Set up tracing in the host binary application.

**Output**

Add the `tracing` and `tracing-subscriber` crates to the `host`'s `Caro.toml`.

```
[dependencies]
...
tracing-subscriber = "0.3.18"
tracing = { version = "0.1.40", default-features = false }
```

Update the host binary application.

```rust
use std::path::PathBuf;
use std::vec::Vec;
use std::{env::current_dir, process};
use clap::Parser;
use alloy_primitives::B256;

#[tokio::main]
async fn main() -> Result<()> {
    let _cli = Cli::parse();

	tracing_subscriber::Registry::default().init();
	tracing::info!("host telemetry initialized");

    Ok(())
}

....
```

**Comment by refcell** [comment]
Closed by #136, great job @N0xMare

**Comment by refcell** [comment]
Completed in #143
